### DISCUSSION TITLE
Schema Update Question

### DISCUSSION THREAD

**Discussion by Murmucka** [CiscoDevNet/iPSK-Manager#59] [Q&A] [answered]
Hi,

i'm to updating sql schema from ver4 to ver5. In the SQL File i need fill data for 2 Variables
<ISE_DB_NAME> is clear, but:

Replace <ISE_DB_USERNAME> with the Database Username created when installing the Database

Wich Username from here (from DONOTDELETE File)
This one:

```
#iPSKManager Database Credentials
#--------------------------------
Host = 127.0.0.1
Username = ipsk-db-user
```

Or This One:
```
#Cisco ISE MySQL Credentials
#---------------------------
Username = ipsk-ise-user
```

Thanks for your help!

**Comment by ciesinsn** [comment, marked as answer]
This one

#iPSKManager Database Credentials
#--------------------------------
Host = 127.0.0.1
Username = ipsk-db-user

**Reply by Murmucka** [reply]
Hi, thanks a lot for helping me, SQL Schema update was successful, but still have message on dashboard

`ALERT: Installation Files are still installed.`

What files i need to remove?

**Reply by ciesinsn** [reply]
Looks like your files may not be owned by the user of the web service do auto delete them.  The files to delete are installer.php and installer.inc.php in the adminportal directory.
